src/BigBlueButton.php

```diff
@@ -549,7 +562,7 @@ private function processXmlResponse(string $url, string $payload = '', string $c
     {
         try {
             return new SimpleXMLElement($this->requestUrl($url, $payload, $contentType));
-        } catch (NetworkException | RuntimeException $e) {
+        } catch (NetworkException|RuntimeException $e) {
```

Comment:
Do you want to keep this spacing?

Reply:
I would use the new style. It's more compact and not harder to read.